app/views/view_data/pages/cookies.html.erb

```diff
@@ -0,0 +1,79 @@
+<main id="content" role="main">
+
+  <div class="grid-row">
+    <div class="column-full">
+      <h1 class="heading-large">
+        Cookies
+      </h1>
+    </div>
+  </div>
+
+  <br/>
```

Comment:
Could get rid of this <br/> as it isn't semantically a line break in a sentence... more for spacing?

Reply:
More for 'forgot to remove it' really.

Reply:
👍